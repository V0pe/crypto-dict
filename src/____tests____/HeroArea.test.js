import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../Components/HeroArea';
import store from '../Redux/Configurestore';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><Header /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
