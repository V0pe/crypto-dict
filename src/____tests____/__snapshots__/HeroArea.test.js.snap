// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <div>
    <button
      className="logoButton"
      onClick={[Function]}
      type="button"
    >
      <h1>
        <img
          alt="logo"
          className="app-logo"
          src="cryptocurrencies.png"
        />
      </h1>
    </button>
  </div>
  <h1 />
</header>
`;
